draft-ietf-gnap-core-protocol.md

```diff
@@ -1968,57 +1788,60 @@ URL that has access to three described resources.
     }
 ~~~
 
-The following non-normative example shows a single access token bound to the client instance's key, which
-was presented using the [detached JWS](#detached-jws) binding method.
+The following non-normative example shows a single bearer access token
+with access to two described resources.
 
 ~~~
     "access_token": {
         "value": "OS9M2PMHKUR64TB8N6BW7OZB8CDFONP219RP1LT0",
-        "key": true,
-        "resources": [
+        "bound": true,
+        "access": [
             "finance", "medical"
         ]
     }
 ~~~
 
 
-If the client instance [requested multiple access tokens](#request-resource-multiple), the AS MUST NOT respond with a
-single access token structure unless the client instance sends the `split_token` flag as described in {{request-resource-flag}}.
+If the client instance [requested multiple access tokens](#request-token-multiple), the AS MUST NOT respond with a
+single access token structure unless the client instance sends the `split` flag as described in {{request-token-single}}.
```

Comment:
On the contrary, if the request has the `split` flag, you can expect the response to contain *more* ATs than the request.

You may have wanted to say (or IMO you should say) that if the client requests multiple ATs, the AS must not return a single AT structure, even if only a single AT is returned.

Reply:
Thanks, I think this was a copy-paste error that needs to be cleaned up!